### ISSUE TITLE
SPARQL result in XML may not be strictly valid?

### ISSUE THREAD

**Issue by pchampin** [RDFLib/rdflib#493]
The XML serializer for SPARQL results produces a result that looks valid to me, but is not recognized by at least one tool (namely [Corese](http://wimmics.inria.fr/corese)). The problem comes from the fact that the serializer produces something like

``` xml
  <sparql:sparql xmlns:sparql="http://www.w3.org/2005/sparql-results#">
```

while Corese expects an empty NS prefix, as in

``` xml
  <sparql xmlns="http://www.w3.org/2005/sparql-results#">
```

which is actually [what the recommendation says](http://www.w3.org/TR/rdf-sparql-XMLres/#docElement). Again, the fact that the documentation uses XML namespaces makes me think that what RDFLib produces is valid, but the phrasing ("written as follow") makes it ambiguous. I explicitly [asked the question on the public mailing list](https://lists.w3.org/Archives/Public/public-sparql-dev/2015JulSep/0007.html).

Whatever the answer is, I guess it wouldn't hurt to be a little more conservative in RDFLib. If I'm correct, changing [that line](https://github.com/RDFLib/rdflib/blob/7f6c9e73c2b7b84fd38811b3064e3740f96d7976/rdflib/plugins/sparql/results/xmlresults.py#L148) to use the empty prefix rather than the "sparql" prefix would do the trick (sorry, not really able to test it myself right now).

**Comment by joernhees** [comment]
@gromgull any feedback on this?

**Comment by joernhees** [comment]
https://lists.w3.org/Archives/Public/public-sparql-dev/2015JulSep/0009.html says it's Corese's problem...

**Comment by gromgull** [comment]
fixed by #588
